### PR TITLE
v4

### PR DESCRIPTION
**Pull request by denza**

### CODE REVIEW

**Comment by denza** [comment]
@edevenport Ready for review.

**Comment by denza** [comment]
@edevenport Ready for review.

**Comment by edevenport** [comment]
@denza I noticed the indentation seems to vary between 4 spaces and tabs. Can we standardize on 4 spaces? Also, can you squash the commits?

@nurfet-becirevic @jasmingacic Can you review?

File: Profitbricks/Server.cs [hunk 1]
```diff
@@ -217,13 +223,22 @@ protected override void BeginProcessing()
                 {
                     Properties = new VolumeProperties
                     {
-                        Size = (int)(size / 1024 / 1024 / 1024),
-                        Image = ImageId,
+                        Size = (int)(size / 1024 / 1024 / 1024),                      
                         Type = (string.IsNullOrEmpty(DiskType) ? "HDD" : DiskType),
-                        Name = Name,
+                        Name = Name,
```
Comment (nurfet-becirevic) [inline Profitbricks/Server.cs]: remove the extra space at the end of the line

File: Profitbricks/Snapshot.cs [hunk 2]
```diff
@@ -252,10 +252,10 @@ public class SetSnapshot : Cmdlet
...
         public bool? DiscScsiHotUnplug { get; set; }
 
-        /// <summary>
-        /// <para type="description">The snapshot's license type: LINUX, WINDOWS, or UNKNOWN.	</para>
-        /// </summary>
-        [Parameter(Position = 13, ValueFromPipeline = true)]
+		/// <summary>
+		/// <para type="description">The snapshot's license type: LINUX, WINDOWS, WINDOWS2016 or UNKNOWN.	</para>
+		/// </summary>
+		[Parameter(Position = 13, ValueFromPipeline = true)]
```
Comment (nurfet-becirevic) [inline Profitbricks/Snapshot.cs]: format spacing

File: Profitbricks/Volume.cs [hunk 3]
```diff
@@ -431,10 +443,10 @@ public class SetVolume : Cmdlet
...
         public string ImageId { get; set; }
 
-        /// <summary>
-        /// <para type="description">The license type of the volume. Options: LINUX, WINDOWS, UNKNOWN, OTHER</para>
-        /// </summary>
-        [Parameter(Position = 6, HelpMessage = "The licence type of the volume. Options: LINUX, WINDOWS, UNKNOWN, OTHER", ValueFromPipeline = true)]
+		/// <summary>
+		/// <para type="description">The license type of the volume. Options: LINUX, WINDOWS, WINDOWS2016, UNKNOWN, OTHER</para>
+		/// </summary>
+		[Parameter(Position = 6, HelpMessage = "The licence type of the volume. Options: LINUX, WINDOWS, WINDOWS2016, UNKNOWN, OTHER", ValueFromPipeline = true)]
```
Comment (nurfet-becirevic) [inline Profitbricks/Volume.cs]: format spacing

**Comment by denza** [comment]
Spacing fixed. Squashed.

**Pull request merged**